### DISCUSSION TITLE
（求助）紧急，怎么换头像和改变字体大小啊？

### DISCUSSION THREAD

**Discussion by fairy-fire** [share121/inter-knot#3039] [其他]
绳网是跟着教程注册的，怎么换头像啊？另外怎么发表情包啊？（只会发图）
![Image_1707375330019](https://github.com/user-attachments/assets/4b69f178-f206-470d-88e9-55f23c4c49ec)

**Comment by isle200inf** [comment]
点你的头像进你的主页，再点你的头像，右面就可以改头像了， 表情包参考 #2049

**Reply by isle200inf** [reply]
或者这里也可以？<https://github.com/settings/profile>
